R/compile_report.R

```diff
@@ -231,15 +234,46 @@ compile_report <- function(file, quiet = FALSE, factory = getwd(),
   }
 
   dir.create(output_dir, FALSE, TRUE)
-
+  
+  output_files <- list()
   for (file in new_files) {
     destination <- file.path(output_dir, file)
     move_file(file, destination)
+    i <- match(file, new_files) #gets index
+    output_files[[i]] <- destination
   }
-
+  
   ## remove empty directories
   to_remove <- file.path(new_dirs)
   file.remove(to_remove)
+  
+  
+  # ================
+  ### ! KEEP AT END OF FUNCTION ! ###
+  ## Logging approach: 
+  ## 1. Combine all env variables and args into a list and add timestamp
+  ## 2. Read exising log file into a dataframe
+  ## 3. Add row to log dataframe and write new csv
+  
+  
+  ## Start log code
+  setwd(odir)
```

Comment:
the log should be at the root of the factory, irrespective of the original folder

Reply:
ah gotchya, i misunderstood